### PR TITLE
New personal item: Jotty's Pet Rock 'Bobby'

### PR DESCRIPTION
**Pull request by Tych0theSynth**
## About the PR
Added a personal item for Joetta-Beedell: Bobby the pet rock. 

## Why / Balance
On request of @luckyshotpictures. Pet rock. PET. ROCK. IS BOBBY.

## Technical details
Tested and working with loadouts. Tested and working in game. 
Delay for animation changes set to 120s.

## Media
https://github.com/Sector-Umbra/Sector-Umbra/assets/151581207/a9fc300b-4051-42f5-aef2-360e9d2c12aa

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

## Breaking changes
I should hope not, unless Jotty throws the rock through a window and kills someone.

**Changelog**
:cl:
- add: Added a pet rock for Jotty.

### CODE REVIEW

File: Resources/Textures/_Umbra/Objects/Fun/bobby.rsi/meta.json [hunk 1]
```diff
@@ -0,0 +1,21 @@
+{
+    "version": 1,
+    "license": "CC-BY-SA-3.0",
+    "copyright": "Taken from Paradise Station, customised by Tych0theSynth",
```
Comment (TsjipTsjip) [inline Resources/Textures/_Umbra/Objects/Fun/bobby.rsi/meta.json:4]: This needs an exact commit reference.

**Review by TsjipTsjip — CHANGES_REQUESTED** [review]
One small thing, rest looks good to go.

File: Resources/Textures/_Umbra/Objects/Fun/bobby.rsi/meta.json (hunk 1 above)
Comment (Tych0theSynth) [reply Resources/Textures/_Umbra/Objects/Fun/bobby.rsi/meta.json:4]: GRAH, the original didn't have one. I'll have to try and find it >.>

**Review by TsjipTsjip — APPROVED** [review]
Bobby.

**Pull request merged**